modules/ROOT/pages/fips-140-2-compliance-support.adoc

```diff
@@ -71,8 +71,17 @@ Provider:bc-fips-1.0.2.4  TLS: bctls-fips-1.0.19.jar  PKIX: bcpkix-fips-1.0.7.ja
 . Register the security provider in the security properties file in the `$JAVA_HOME/conf/security/java.security` folder so that only the Bouncy Castle providers are set.
 . Copy the 3 files downloaded to `/lib/boot` (needed for Mule runtime) and `/mule-agent-plugin/lib` (needed for Mule agent) folders in Mule runtime.
 . Run `amc_setup` and register Mule runtime. For example, in *Add Server*, you find the registration token `/amc_setup -H d32c4bde-7184-444a-bd87-7a274623e14f---704247 node10  --fips`, where `d32c4bde-7184-444a-bd87-7a274623e14f---704247` is the key from Anypoint Runtime Manager.
++
+When configuring Mule Agent, include the `--fips` flag to enable FIPS compliance. If you're using Government Cloud, also include the `--region us_gov` flag:
```

Comment:
Is it "Mule Agent" or "Mule agent"? I see "Mule agent" on line 72

Reply:
Mule Agent plugin. I'll fix the other entry.